serde/de.py

```diff
@@ -437,7 +470,14 @@ def render(self, arg: DeField) -> str:
             elif has_default_factory(arg):
                 res = f'({res}) if {exists} else serde_scope.defaults["{arg.name}"]()'
 
-        return res
+        if self.custom and not arg.deserialize:
+            # The function takes a closure in order to execute the default value lazily.
+            return (
+                f'serde_custom_class_deserializer({arg.type.__name__}, {arg.datavar}, {arg.data}, '
+                f'default=lambda cls, data: {res})'
```

Comment:
Why do you pass arguments to the lambda here?
I think a lambda without arguments would work fine and is cleaner.

Reply:
You're right. It works 😄